### PR TITLE
Move admin related docs into docs/admin

### PR DESCRIPTION
**Pull request by lavalamp**
Still working on this.

cc @erictune @bgrant0607

### CODE REVIEW

**Comment by lavalamp** [comment]
@davidopp @bgrant0607 for file movement changes (I also split namespaces.md into user/admin versions)

@thockin @erictune for mungedocs changes.

**Comment by bgrant0607** [comment]
Changes seem reasonable.

We'll need a user-oriented networking doc, and maybe bits of some of the access docs as well, but I'm fine with the move.

File: cmd/mungedocs/links.go [hunk 1]
```diff
@@ -126,18 +144,49 @@ func checkPath(filePath, linkPath string) (newPath string, ok bool) {
...
+		if info.IsDir() {
+			return linkPath + "/", true
+		}
+		return linkPath, true
+	}
+
+	for strings.HasPrefix(linkPath, "../") {
+		linkPath = strings.TrimPrefix(linkPath, "../")
+	}
+
```
Comment (erictune) [inline cmd/mungedocs/links.go]: Add comment that this is searching for broken link targets and fixing them.

**Comment by erictune** [comment]
LGTM

**Comment by bgrant0607** [comment]
docs/accessing-the-cluster.md is sort of (or partly) a user doc.

 docs/accessing_the_api.md is sort of (or partly) a developer doc.

**Comment by bgrant0607** [comment]
Proposal to move user docs: #11093

**Comment by bgrant0607** [comment]
Ref #7299

**Comment by bgrant0607** [comment]
Re. link munging: we have a number of links through releases.k8s.io, including from the swagger descriptions #10854

**Comment by thockin** [comment]
status of this wrt the other munger PR?

**Comment by bgrant0607** [comment]
I'd like to get this in. Could we separate the moving from the munging? 

cc @mikedanese

**Comment by erictune** [comment]
look forward to this going in so I can build on it.

**Comment by lavalamp** [comment]
Sorry for delay; rebased. Should be good now.

**Comment by bgrant0607** [comment]
LGTM
ok to merge

**Pull request merged**